paper/paper.md

```diff
@@ -0,0 +1,140 @@
+      index: 3
+    - name: Independent Contributor
+      index: 4
+
+date: 29 October 2021
+bibliography: paper.bib
+---
+
+# Summary
+
+Cities, counties, and states throughout the USA are bound by law to archive recordings of public meetings. Most local governments comply with these laws by posting documents, audio, or video recording online. Council Data Project (CDP) is three open-source tools that improve the accessibility of local government data by systematically collecting, transforming, and re-publishing this data to the web. The software provided here can be used by civic technologists to promote government transparency, or in support of research by using processed data for public policy, natural language processing and machine learning, or information retrieval and discovery.
+
+# Statement of Need
+
+Comparative research into municipal governance in the USA is often prohibitively difficult due to a broad federal system where states, counties, and cities divide legislative powers differntly. This has contributed to the lack of large-scale quantitative studies of municipal government, and impeded necessary research into  effective procedural elements of administrative and legislative processes [@Trounstine:2009]. Council Data Project (CDP) enables large-scale quantitative studies by generating standardized municipal governance corpora - inclduing X, Y, and Z.
+
+## Related Work
+
+Work in extracting and repackaging government data into machine-readable and experiment ready datasets has historically happened in fields with highly structured data, such as meteorology [@Sparks2017] and legal review and monitoring [@courtlistener]. Notably, there has been prior work in extracting and repackaging municipal government data with [Councilmatic](https://github.com/codeforamerica/councilmatic) [@councilmatic]. However, this work largely aimed to make municipal data more accessible to a general public, and didn't add any specific data processing to expand the research capabilities of the produced dataset. Recent advances in natural language processing have made it possible to conduct large-scale transcript-based studies on the effects of gender, ideology, and seniority in Supreme Court oral argument [@jacobi2017] and the effects that information communication technology has on civic participation [@einstein2021].
+
+## CDP Architecture
+
+Council Data Project consists of three primary tools:
+
+1. [cookiecutter-cdp-deployment](https://github.com/CouncilDataProject/cookiecutter-cdp-deployment): A Python [cookiecutter](https://cookiecutter.readthedocs.io/) [@cookiecutter] template to assist users in fully deploying a new CDP instance. A "CDP Instance" is a unique deployment of CDP software and tools. For example, there is an "instance" of CDP for the ["Seattle City Council"](https://councildataproject.org/seattle/#/) and an instance of CDP for the "King County Council". Each instance is comprised of it's own repository and infrastructure.
```

Comment:
>Each instance is comprised of it's own repository and infrastructure.

Be more specific about what the infrastructure contains - does it contain `cdp-backend` ?